filebeat/_meta/fields.common.yml

```diff
@@ -108,6 +108,10 @@
       description: >
         Logging level.
 
+    - name: log.status
```

Comment:
What about calling it `log.flags`? I somehow expect a different content for status like `unprocessed` or `processed`. `flags` is inspired by your field description and I kind of like it because we flagged a certain event.

Alternative would be `log.tags`?